### PR TITLE
Adding tests for core/ast_transforms/processor.py

### PR DESCRIPTION
**Pull request by AaronCritchley**
The 99% coverage was bothering me more than it should, so I opted to add some tests for the processor file.

I plan on hitting some of the TODO's in `AstProcessor` in the future so will add more comprehensive testing then.

### CODE REVIEW

**Comment by AaronCritchley** [comment]
I don't think the failed travis build is my fault, it looks like it stalled on `tests/optimize/root_finding/test_newton_raphson_hypo.py`. Let me know if I've done something and screwed the build though.

File: tests/core/ast_transforms/test_processor.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+from fastats.core.ast_transforms.processor import recompile, uncompile
+
+
+def test_recompile_happy_path():
+    func_as_string = '''
+def f(n):
+     # This is a comment
+     return n ** 2
+'''
```
Comment (pawroman) [inline tests/core/ast_transforms/test_processor.py]: The zero indent bothers me slightly. Do you think we could use [`textwrap.dedent`](https://docs.python.org/3.5/library/textwrap.html#textwrap.dedent) here and in the function below?

Comment (AaronCritchley) [reply tests/core/ast_transforms/test_processor.py]: No problem - implemented.

**Comment by pawroman** [comment]
Nice work, thanks!

**Pull request merged**